inc/c_util/async_op.h

```diff
@@ -56,6 +56,7 @@ typedef struct ASYNC_OP_TAG
 THANDLE_TYPE_DECLARE(ASYNC_OP);
 
 MOCKABLE_FUNCTION(, THANDLE(ASYNC_OP), async_op_create, ASYNC_OP_CANCEL_IMPL, cancel, uint32_t, context_size, uint32_t, context_align, ASYNC_OP_DISPOSE, dispose);

```

Comment:
>uint32_t, context_size, uint32_t, context_align, [](http://example.com/codeflow?start=86&length=48)

consider: since I'm looking at this again, I was thinking at one point a macro wrapper on top of this could be nice to make it less error prone:

```c
#define ASYNC_OP_CREATE(T)(cancel, dispose) \
    async_op_create(cancel, sizeof(T), alignof(T), dispose)
```

Reply:
[https://msazure.visualstudio.com/One/_workitems/edit/30417987](https://msazure.visualstudio.com/One/_workitems/edit/30417987)